cookies/rfc6265bis/index.html

```diff
@@ -0,0 +1,364 @@
+<!DOCTYPE html>
```

Comment:
We don't need this file. It made sense for the stand-alone site, it doesn't make sense as part of WPT.